pages/linux/calcurse.md

```diff
@@ -0,0 +1,23 @@
+#calcurse
+
+> A text-based calendar and scheduling application for the command line.
+
+- Start calcurse on interactive mode:
+
+`calcurse`
+
+- Print the appointments and events for the current day and exit:
+
+`calcurse -a` or `calcurse --appointment`
```

Comment:
Perhaps this will read better? We generally prefer the long option only, to make it clearer.

```suggestion
`calcurse --appointment`
```

Advanced users can always look up the shortened version :-)